products/compute/terraform.yaml

```diff
@@ -162,6 +162,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           regex: '^(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)$'
       cdnPolicy: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+      cdnPolicy.cacheMode: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
```

Comment:
Could we set explicit defaults instead? Those are generally preferable to `default_from_api`, as they'll show up at plan time.

Reply:
Google doesn't set one.

Also, setting signedUrlCacheMaxAgeSec makes Google set cachemode but when you set other fields that field is null. 

Funny API Contract and the docs don't mention the default one either

Reply:
Thanks! `default_from_api` makes sense, in that case.